terraform/virtual-network.tf

```diff
@@ -58,6 +65,14 @@ resource "azurerm_virtual_network_peering" "vh_infra_core_ado_TO_hubs" {
   provider                  = azurerm.current_sub_peer
 }
 
+resource "azurerm_virtual_network_peering" "core-infra-vnet-mgmt_TO_vh_infra_core_ado" {
+  name                      = "core-infra-vnet-mgmt-TO-vh-infra-core-ado"
```

Comment:
can these be variables e.g. "${var.core_infra_vnet_name}-TO-${var.ado_vnet_name}"